packages/hardhat-verify/src/internal/etherscan.ts

```diff
@@ -97,7 +97,7 @@ export class Etherscan {
     url.search = parameters.toString();
 
     const response = await sendGetRequest(url);
-    const json = await response.body.json();
+    const json: any = await response.body.json();
```

Comment:
are these lines working? Because the `EtherscanResponse` constructor takes an `any`.

```
const etherscanResponse = new EtherscanResponse(await response.body.json());
```

https://github.com/NomicFoundation/hardhat/pull/4293/files#diff-87b22bea7502ad3ea606dc40db78f8e4e4837e1fcb849bb3ffa183976a4cd0f6L163

https://github.com/NomicFoundation/hardhat/pull/4293/files#diff-87b22bea7502ad3ea606dc40db78f8e4e4837e1fcb849bb3ffa183976a4cd0f6L216

Reply:
Yes beause you can assign anything to an `any` value, including an `unknown` value.